modules/protobuf-matchers/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/inazarenko/protobuf-matchers",
+    "maintainers": [
+        {
+            "email": "igor.n.nazarenko@gmail.com",
+            "github": "inazarenko",
```

Comment:
@inazarenko Are you okay with being a maintainer of this module?

Reply:
Yep. I will need to figure out how to update the registry when I make changes, but TBH this code is pretty static so there's not a lot of maintenance.